compiler/rustc_lint/src/internal.rs

```diff
@@ -427,7 +427,7 @@ declare_tool_lint! {
     /// More details on translatable diagnostics can be found
     /// [here](https://rustc-dev-guide.rust-lang.org/diagnostics/translation.html).
     pub rustc::UNTRANSLATABLE_DIAGNOSTIC,
-    Deny,
+    Allow,
```

Comment:
Please downgrade both this and the other lint too, right after. Otherwise you still can't use the old style diagnostics in the compiler lol

Reply:
Oh oops, good catch, I'll downgrade the other one too.

Reply:
Downgraded the `diagnostic_outside_of_impl` lint to `allow` as well. Updated PR title/description to match.